web/client/components/map/openlayers/VectorStyleNew.js

```diff
@@ -0,0 +1,255 @@
+const {isNil, trim, isString, isArray, castArray, head, last, find} = require('lodash');
```

Comment:
Rename this VectorStyle, if you have to keep the old one, rename it Legacy, or something like that